src/Http/Place/LegacyPlaceRequestBodyParser.php

```diff
@@ -28,12 +28,20 @@ public function parse(ServerRequestInterface $request): ServerRequestInterface
         }
 
         if (isset($data->type) && $data->type instanceof stdClass) {
+            $terms = [
+                'id' => $data->type->id,
```

Comment:
`type.id` is technically required but not guaranteed to be set since we don't actually validate the JSON before this step. The JSON schema validation only happens after this transformation of the legacy format.